prime-router/src/main/kotlin/cli/tests/AuthTests.kt

```diff
@@ -1294,4 +1295,25 @@ fun lookupTableReadAndWriteSmokeTests(
         failedMessages += "Got empty LookupTable List."
     }
     return Pair(passed, failedMessages)
-}
\ No newline at end of file
+}
+
+/**
+ * The access token left by a previous login command as specified by the command line parameters
+ * @param environment Where is the test hitting? Staging, Local?
+ * @param testName Name of test for logging
+ * @return The access token if we're in the oktaApp environment, otherwise "dummy"
+ */
+fun getOktaAccessToken(
```

Comment:
Should this live maybe in OktaAuthTests?  It's a little confusing since there are a bunch of these with close to the same name.

Reply:
Will combine these auth functions into one